Main1.java

```diff
@@ -0,0 +1,15 @@
+package task1;
+
+import java.util.ArrayList;
+
+public class Main1 {
+    public static void main(String[] args) {
+        Numbers numbers = new Numbers( new ArrayList<>());
+        numbers.whileLoopAddNumbers();
```

Comment:
Maybe the naming of this method could be different (without specifying whileLoop in the name :)). 
For example:
```
addNumbersBasedOnUsersInput()
```